### PR TITLE
Specify custom error message in check_func

### PR DESCRIPTION
**Pull request by whysoserious**
I missed that small feature in your awesome library, so there you go!

### CODE REVIEW

**Comment by madeinussr** [comment]
@whysoserious sorry, I didn't notice your PR until merged mine with exact feature. But, if you will deal with merge conflicts (you can wipe out my implementation and test, as they pretty same) and will push your changes - I'll merge your PR with respect of your work!

File: lib/exop/validation_checks.ex [hunk 1]
```diff
@@ -299,10 +299,11 @@ defmodule Exop.ValidationChecks do
   def check_func(check_items, item_name, check) do
     check_item = get_check_item(check_items, item_name)
 
-    if check.(check_item) do
-      true
-    else
-      %{item_name => "isn't valid"}
+    case check.(check_item) do
+      {:error, msg} -> %{item_name => msg}
+      false -> %{item_name => "isn't valid"}
```
Comment (madeinussr) [inline lib/exop/validation_checks.ex]: https://github.com/madeinussr/exop/blob/master/lib/exop/validation_checks.ex#L305

you can change this with your implementation.

**Comment by madeinussr** [comment]
...and bump version to 0.4.1 after

**Comment by madeinussr** [comment]
Ok, I have handled it. Reverted my changes and merged yours. So, the order was kept.

**Comment by whysoserious** [comment]
Thanks as well!

**Pull request merged**